tsconfig.json

```diff
@@ -8,14 +8,25 @@
     "removeComments": false,
     "sourceMap": false,
     "target": "ES5",
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "lib": [
```

Comment:
What is this?

Reply:
this is for better es5 support
for example in src/devtools/devtoolsMiddleware.ts:17 find method is added in es6
i only added the "es2015.core" but if not defined ts add automatically 
► For --target ES5: DOM,ES5,ScriptHost

Reply:
more information in:
https://www.typescriptlang.org/docs/handbook/compiler-options.html#compiler-options
(lib option)